_config.yml

```diff
@@ -71,14 +68,11 @@ navigation:
     - title: Issues
       url: https://github.com/xenia-project/xenia/issues
       icon: bug_report
-#    - title: Buildbot
-#      url: http://build.xenia.jp/waterfall
-#      icon: settings_system_daydream
 
 artifacts:
-  - title: Windows 7+ x64
+  - title: Windows 10 x64
```

Comment:
This is factually incorrect. It's Windows 7 or up. The title of the branch (later) clarifies why, well, that does it inaccurate too. If you want to reduce the noise here, then move the information from the branch title to the artifact title.

Reply:
What do you think of this? https://margen67.github.io/xenia/download/